entrypoint.sh

```
@@ -2,12 +2,16 @@
 echo "🚀 Starting VinaAcademy Frontend Container..."
 
 # Validate required environment variables
-if [ -z "$NEXT_PUBLIC_API_URL" ] || [ -z "$NEXT_PUBLIC_SITE_URL" ] || [ -z "$NEXT_PUBLIC_WS_URL" ]; then
+if [ -z "$NEXT_PUBLIC_API_URL" ] 
```

Shellcheck (0.11.0)

[error] Couldn't parse this if expression. Fix to allow more checks.

(SC1073)